dev/com.ibm.ws.springboot.support.shutdown/src/com/ibm/ws/springboot/support/shutdown/FeatureAuditor.java

```diff
@@ -67,7 +67,7 @@ public void postProcessEnvironment(ConfigurableEnvironment env, SpringApplicatio
         try {
             // Not sure if this is correct....
             Class.forName("org.springframework.boot.web.servlet.server.ServletWebServerFactory");
-            Class.forName("com.ibm.ws.springboot.support.web.server.version30.container.LibertyConfiguration");
+            Class.forName("io.openliberty.springboot.support.web.server.version30.container.LibertyConfiguration");
```

Comment:
How is LibertyConfiguration loadable if it is not exported by the springboot.support bundle?

Reply:
A service ref declared in com.ibm.ws.app.manager.springboot is resolved to  springBoot30Server.jar, and those packages are declared private but available to that FeatureAuditor class